open-company/open-company-web
# Data topic editing

card: https://trello.com/c/hxVPPU6v

To test Finances:
- edit cash
- edit costs
- edit revenue
- save the new data
- mix the previous actions
- check that the chart always respect the latest data inserted

To test growth:
- try to edit the data of an existing metric
- try to edit the metadata of an existing metric
- try to add a new metric
- update the new inserted metric after it was saved
- try to remove a metric
